people/bedroesb/rdmkit/README.md

```diff
@@ -0,0 +1,15 @@
+# /rdmkit/
+This [W3ID](https://w3id.org) provides a persistent URI namespace for RDMkit resources.
+
+## Uses
+This namespace will redirect all urls from https://w3id.org/rdmkit/**ID** to https://bedroesb.github.io/rdmkit/**ID** 
```

Comment:
sorry, but this is not accurate, is it? The w3id would be w3id.org/people/bedroesb/rdmkit/id